cdk-project/lib/images/codebuild-image/python/src/notebooks/parse.py

```diff
@@ -225,3 +258,60 @@ def skip(notebook):
     elif any([str(directory) in SKIP_LIST for directory in directories]):
         return True
     return False
+
+
+def local_mode_mandatory(notebook):
+    """Check whether the notebook should be skipped because runs only in Local Mode
+
+    Args:
+        notebook (Path): The notebook to check.
+
+    Returns:
+        bool: True if local mode is mandatory.
+
+    """
+    directories = Path(notebook).parents
+    return "reinforcement_learning" in directories and str(notebook) not in get_lm_optional_nb_names()
```

Comment:
is this only for "reinforcement_learning", "get_lm_optional_nb_names" has some files with "frameworks" in it

Reply:
Will update